src/components/onboarding.tsx

```diff
@@ -102,8 +102,8 @@ export default function Onboarding() {
 				<GradientBox />
 			</div>
 
-			<div className="w-1/2 bg-white p-[50px] flex flex-col app-no-drag-region">
-				<div className="h-[569px] flex flex-col justify-center items-start flex-[1_0_0%] gap-8">
+			<div className="w-1/2 bg-white p-[50px] flex flex-col">
+				<div className="h-[569px] flex flex-col justify-center items-start flex-[1_0_0%] gap-8 app-no-drag-region">
 					<div className="flex flex-col items-start self-stretch gap-6">
```

Comment:
I noted on the Onboarding screen that there is some non-draggable whitespace above and below the Add Site form. It seems that if the `app-no-drag-region` class were moved one more element further down, the areas in purple would be draggable as well, still leaving the form itself interactable. I wasn't sure if leaving this space non-draggable was intentional, however, so disregard if so.

```
<div className="flex flex-col items-start self-stretch gap-6  app-no-drag-region">
```

<img width="951" alt="Screenshot 2024-05-02 at 12 01 38 pm" src="https://github.com/Automattic/studio/assets/643285/c97133b1-eff5-4c82-a98f-3a1a07217712">

<details>
<summary>Example</summary>


https://github.com/Automattic/studio/assets/643285/c8300e61-a7a1-4044-b958-efe3b915d88f

</details>

Reply:
That's a good thought, I've gone ahead to implement your suggestion in https://github.com/Automattic/studio/pull/75/commits/f8a790db98a2e156d6a0a5e9f88f0c022edbc553. 

I had been a bit conservative with the amount of "draggable space" I was introducing, but agree that for the onboarding screen, it would feel a bit odd for all of that white space to not be draggable while the blue sidebar is completely draggable.